### PR TITLE
Feature/add stim300 driver

### PR DESCRIPTION
**Pull request by VNaas**
Stim300 works

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,30 +1,43 @@
...
+    net-tools \     
+    tcpdump 
 
 RUN echo "source /opt/ros/noetic/setup.bash" >> /home/vortex/.bashrc
 RUN echo "source /home/vortex/asv_ws/devel/setup.bash" >> /home/vortex/.bashrc
 
 RUN mkdir -p /home/vortex/asv_ws
 RUN chown vortex /home/vortex/asv_ws
+RUN chmod a+rw /dev/ttyUSB0
+RUN chmod a+rw /dev/ttyUSB1
```
Comment (chrstrom) [inline Dockerfile:42]:
```suggestion
RUN chown vortex /home/vortex/asv_ws
RUN chmod a+rw /dev/ttyUSB*

```

Will give permission to all currently connected USB devices

**Review by chrstrom — CHANGES_REQUESTED** [review]
Nice! I like the Vortex-AUV style dependencies setup. Just a minor change req before accepting

**Review by ronjakr — APPROVED** [review]
lgtm

**Review by chrstrom — APPROVED** [review]
lgtm

**Pull request merged**